pkg/distributor/forwarding/forwarding.go

```diff
@@ -59,6 +59,14 @@ func NewForwarder(cfg Config, reg prometheus.Registerer, log log.Logger) Forward
 		pools: newPools(),
 		log:   log,
 		reqCh: make(chan *request, cfg.RequestConcurrency),
+		client: http.Client{
+			Transport: &http.Transport{
+				MaxIdleConns:        0,                      // no limit
+				MaxIdleConnsPerHost: cfg.RequestConcurrency, // if not set, it defaults to 2
```

Comment:
The comment is a bit confusing because the default value for `cfg.RequestConcurrency` is `10`, you mean the default for `MaxIdleConnsPerHost`, right?

Reply:
Yes, comment is about `MaxIdleConnsPerHost`, I will update it.